scripts/sync-package-versions.js

```diff
@@ -0,0 +1,64 @@
+const { readFileSync, writeFileSync } = require('fs');
+const { join } = require('path');
+const { execSync } = require('child_process');
+
+fileProcessCloud = './lib/process-services-cloud/package.json'
+
+const processCloudContents = readFileSync(fileProcessCloud).toString();
```

Comment:
Nice. Can we have a more generic solution? :P

Reply:
nope